ext/hal/st/stm32cube/stm32f4xx/drivers/src/stm32f4xx_hal_eth.c

```diff
@@ -168,6 +168,20 @@ static void ETH_Delay(uint32_t mdelay);
   * @{
```

Comment:
Typically we try to avoid editing of the HAL files as these would be overwritten by a new version. Is it possible to get a new version of the STM32 HAL that would contain these changes already?

Reply:
I'm not aware of such implementation right now.